src/commands/advanced_generate.ts

```diff
@@ -660,7 +661,7 @@ ETA: <t:${Math.floor(Date.now()/1000)+(status?.wait_time ?? 0)}:R>`
                 if(!isNaN(Number(option.value)) && option.value) {
                     const lora_by_id = await context.client.fetchLORAByID(option.value, context.client.config.advanced_generate?.user_restrictions?.allow_nsfw)
 
-                    if(lora_by_id?.name) ret.push({
+                    if(lora_by_id?.name && (lora_by_id?.modelVersions[0]?.files[0]?.sizeKB && lora_by_id?.modelVersions[0]?.files[0]?.sizeKB <= 220000 || context.client.horde_curated_loras?.includes(lora_by_id.id))) ret.push({
```

Comment:
i'd add parenthesis around the A || B to make it more clear which ones belong together

Reply:
Done, but I did it wrong. I was thinking that if it's in the curated list then it doesn't matter if there's a file size, but I think if there's no file size then that means there's no file available (not sure if that's actually possible, tbh) in which case it doesn't matter if it's in the curated list. I'll switch the parentheses around.

Reply:
Done